external-api/src/cronJobs/eStoreFacilityFolderCreationJob.cron.ts

```diff
@@ -12,7 +12,7 @@ export const eStoreFacilityFolderCreationJob = async (eStoreData: Estore) => {
     const response = await cronJobLogsCollection.findOneAndUpdate(
       { dealId: { $eq: eStoreData.dealId } },
       { $inc: { facilityFolderRetries: 1 } },
-      { returnNewDocument: true, returnDocument: 'after' },
+      { returnDocument: 'after' },
```

Comment:
The `returnNewDocument` option does not exist in `mongodb 4.17.0` and typescript was throwing some errors with it, so I have removed it

Reply:
I believe it was `5.Y.Z` (can't remember minor version)

Reply:
[This link](https://mongodb.github.io/node-mongodb-native/3.6/api/Collection.html#findOneAndUpdate) shows what options are available, and `returnNewDocument` is not there. It may have been in an old version of MongoDB, but even then (as can be seen in the documentation), it seems like the option to return a document should be entirely controlled by the `returnDocument` parameter (even `returnOriginal` is now deprecated!)